evalml/tests/component_tests/test_undersampler.py

```diff
@@ -56,7 +56,7 @@ def test_undersample_imbalanced(data_type, make_data_type):
 
     assert len(new_X) == 750
     assert len(new_y) == 750
-    assert all(new_y.to_series().value_counts().values == [600, 150])
+    pd.testing.assert_series_equal(new_y.to_series().value_counts(), pd.Series([600, 150], index=[1, 0]), check_dtype=False)
```

Comment:
I think I meant here something similar to 
```
class_imbalance = <some calculation>
assert class_imbalance == 4
```

Reply:
Ah, I get it. I can add that in